_includes/post-categories.html

```diff
@@ -0,0 +1,10 @@
+<span class="post-categories">{%- if page.categories.size == 1 -%}Category{%- else -%}Categories{%- endif -%}:&nbsp;
+  {%- for category in page.categories -%}
+    {%- if site.minima.archive_page.categories -%}
+      <a href="{{ site.minima.archive_page.categories }}/{{ category | slugify }}/">{{ category }}</a>
+    {%- else -%}
+      {{ category }}
+    {%- endif -%}
+    {%- if forloop.last == false -%},&nbsp;{%- endif -%}
```

Comment:
use Liquid `unless` instead
```suggestion
    {% unless forloop.last %},&nbsp;{% endunless %}
```

Reply:
OK